apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -337,36 +256,33 @@ public IOFileFilter getDirectoriesToIgnore() {
      * @param directoriesToIgnore the filter that defines the directories to ignore.
      * @see Defaults#DIRECTORIES_TO_IGNORE
      */
-    public void setDirectoriesToIgnore(final IOFileFilter directoriesToIgnore) {
+    public void setDirectoriesToIgnore(IOFileFilter directoriesToIgnore) {
         this.directoriesToIgnore = directoriesToIgnore == null ? FalseFileFilter.FALSE : directoriesToIgnore;
     }
 
     /**
      * Adds a directory filter to the directories to ignore.
      * @param directoryToIgnore the directory filter to add.
      */
-    public void addDirectoryToIgnore(final IOFileFilter directoryToIgnore) {
+    public void addDirectoryToIgnore(IOFileFilter directoryToIgnore) {
         this.directoriesToIgnore = this.directoriesToIgnore.or(directoryToIgnore);
     }
 
     /**
-     * Gets the reportable object.
      * @return the thing being reported on.
      */
     public IReportable getReportable() {
         return reportable;
     }
 
     /**
-     * Sets the reportable object.
      * @param reportable the thing being reported on.
      */
-    public void setReportable(final IReportable reportable) {
+    public void setReportable(IReportable reportable) {
         this.reportable = reportable;
     }
 
     /**
-     * Gets the IOSupplier with the style sheet.
```

Comment:
If I'm not too mistaken this will result in more javadoc warnings about missing javadocs? Is that intended?